lamindb/models/record.py

```diff
@@ -671,75 +671,86 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         skip_validation = kwargs.pop("_skip_validation", False)
-        if not args and skip_validation:
-            super().__init__(**kwargs)
-        elif not args and not skip_validation:
-            from ..core._settings import settings
-            from .can_curate import CanCurate
-            from .collection import Collection
-            from .schema import Schema
-            from .transform import Transform
-
-            validate_fields(self, kwargs)
-
-            # do not search for names if an id is passed; this is important
-            # e.g. when synching ids from the notebook store to lamindb
-            has_consciously_provided_uid = False
-            if "_has_consciously_provided_uid" in kwargs:
-                has_consciously_provided_uid = kwargs.pop(
-                    "_has_consciously_provided_uid"
-                )
+        if not args:
             if (
-                isinstance(self, (CanCurate, Collection, Transform))
-                and settings.creation.search_names
-                and not has_consciously_provided_uid
+                issubclass(self.__class__, Record)
+                and not self.__class__.__name__ == "Storage"
+                # do not save bionty entities in restricted spaces by default
+                and self.__class__.__module__ != "bionty.models"
```

Comment:
This looks pretty bad to me actually. Is it even documented? An even if it is, stacking such exceptions is not a good thing.

Reply:
What do you suggest?

>  Is it even documented?

No, we should debate whether we want to do it. But my gut feeling is that this is the right default; we only want to write into a dedicate space in bionty if the user is very explicit about it. Otherwise I expect that many things in bionty end up in a restricted space by accident.

Reply:
I guess this is fine if we make a note somewhere that `bionty` models ignore this settings. By explicit you mean manually assigning space to an object, right?

Reply:
Yes. I think users should manually set a space on a bionty object rather than doing this through `ln.track()` for "all objects created in a session".

Reply:
Maybe a remark somewhere about this behavior?

Reply:
Ok, am adding this to the `ln.track()` documentation and guide.

Reply:
Thx.